cmd/openstack-populator/openstack-populator.go

```diff
@@ -97,8 +104,25 @@ func loadConfig(secretName, endpoint, namespace string) openstackConfig {
 	}
 }
 
-func populate(crName, crNamespace, fileName, endpoint, secretName, imageID string) {
-	config := loadConfig(secretName, endpoint, crNamespace)
+func populate(fileName, endpoint, secretName, imageID, controllerIp string) {
+	http.Handle("/metrics", promhttp.Handler())
+	go http.ListenAndServe(":2112", nil)
+	progressGague := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: "volume_populators",
+			Name:      "openstack_volume_populator",
+			Help:      "Amount of data transferred",
+		},
+		[]string{"image_id"},
```

Comment:
here it's image_id, ovirt will use something else and v2v already using disk_id (or planning to be). it would be nice if we could have a something single to align in the forklift-controller in order to read.

Reply:
disk_id doesn't really make sense for openstack, we can revisit this when it's actually unified